doc/installation.rst

```diff
@@ -1,20 +1,20 @@
-How to Install |PyInstaller|
+How to Install PyInstaller
 ===============================
 
-|PyInstaller| is a normal Python package.
+PyInstaller is a normal Python package.
 You can download the archive from PyPi_,
 but it is easier to install using pip_ where is is available,
 for example::
 
-    pip install pyinstaller
+    pip install PyInstaller
 
 or upgrade to a newer version::
 
-    pip install --upgrade pyinstaller
+    pip install --upgrade PyInstaller
 
 To install the current development version use::
 
-    pip install https://github.com/pyinstaller/pyinstaller/tarball/develop
+    pip install https://github.com/PyInstaller/PyInstaller/tarball/develop
```

Comment:
How about `pip install git+https://github.com/PyInstaller/PyInstaller`?

(and perhaps mention `pip install git+https://github.com/PyInstaller/PyInstaller@branch`)

Reply:
Will deal with this in a later PR.